packages/webapi/admapi/committeerecord.go

```diff
@@ -91,7 +93,13 @@ func (s *committeeRecordService) handleGetCommitteeForChain(c echo.Context) erro
 	if err != nil {
 		return httperrors.BadRequest(err.Error())
 	}
-	chain := s.chains().Get(chainID)
+	inCludeInActive, err := strconv.ParseBool(c.QueryParam("includeInActive"))
```

Comment:
```suggestion
	inCludeInActive, err := strconv.ParseBool(c.QueryParam("includeInActive"))
```

there is some funky capitalization going on... `inCludeInActive` is just a typo I think
`includeInActive` - means "include in active"? I think we wanted "include inactive" or "includeDeactivated"?